frontend/packages/ux-editor/src/App.tsx

```diff
@@ -31,36 +35,29 @@ export function App() {
   const t = useText();
   const { org, app } = useParams();
   const [searchParams, setSearchParams] = useSearchParams();
-  const layoutPagesOrder = useSelector(
-    (state: IAppState) => state.formDesigner.layout.layoutSettings.pages.order
-  );
-  const layoutOrder = useSelector(
-    (state: IAppState) =>
-      state.formDesigner.layout.layouts[state.formDesigner.layout.selectedLayout]?.order
-  );
-
-  const selectedLayout = useSelector(
-    (state: IAppState) => state.formDesigner.layout.selectedLayout
-  );
-
-  const activeList = useSelector((state: IAppState) => state.formDesigner.layout.activeList);
 
   const datamodelQuery = useDatamodelQuery(org, app);
+  const formLayoutsQuery = useFormLayoutsQuery(org, app);
```

Comment:
Dette er smak og behag, men hva tenker du om å gjøre det sånn som dette? :)
```suggestion
  const { data: formLayouts, isError: isFormLayoutsError  } = useFormLayoutsQuery(org, app);
```

Reply:
Ikke dumt!